rules/bicep-do-you-use-user-defined-data-types/rule.md

```diff
@@ -3,7 +3,7 @@ type: rule
 archivedreason:
 title: Bicep - Do you use User-defined Data Types?
 guid: 2a3ba816-ba40-4cfe-8193-7e151e038ac5
-uri: bicep-user-defined-data-types
+uri: bicep-do-you-use-user-defined-data-types
```

Comment:
Hey @ricksu978 

I would do the other way around... fixing the folder instead, so we follow https://www.ssw.com.au/rules/create-friendly-short-urls/

1. Please action (let me know on Teams if unsure how to proceed)